_includes/graphql/getting-started.md

```diff
@@ -21,3 +21,79 @@ After running the CLI command, you should have something like this in your termi
 Since you have already started your Parse GraphQL Server, you can now visit [http://localhost:1337/playground](http://localhost:1337/playground) in your web browser to start playing with your GraphQL API.
 
 <img alt="GraphQL Playground" data-echo="{{ '/assets/images/graphql/graphql-playground.png' | prepend: site.baseurl }}"/>
+
+## Using Docker
+
+You can also run the Parse GraphQL API inside a Docker container:
+
+```bash
+$ git clone https://github.com/parse-community/parse-server
+$ cd parse-server
+$ docker build --tag parse-server .
+$ docker run --name my-mongo -d mongo
+$ docker run --name my-parse-server --link my-mongo:mongo -d parse-server --appId APPLICATION_ID --masterKey MASTER_KEY --databaseURI mongodb://mongo/test --mountGraphQL --mountPlayground
+```
+
+After starting the server, you can visit [http://localhost:1337/playground](http://localhost:1337/playground) in your browser to start playing with your GraphQL API.
+
+Note:
+* ⚠️ Please do not use `--mountPlayground` option in production as anyone could access your API Playground and read or change your application's data. [Parse Dashboard](#running-parse-dashboard) has a built-in GraphQL Playground and it is the recommended option for production apps.
```

Comment:
I don't think it makes sense to have a bulleted list for one item.

Reply:
<img width="548" alt="Screenshot 2019-07-14 at 21 01 40" src="https://user-images.githubusercontent.com/13188249/61188689-eafa3d80-a67a-11e9-84c8-f12318240dac.png">

Reply:
done